examples/ct/handlers.go

```diff
@@ -337,20 +338,20 @@ func getSTH(c LogContext, w http.ResponseWriter, r *http.Request) (int, error) {
 	}
 
 	// Now build the final result object that will be marshalled to JSON
-	jsonResponse := getSTHResponse{
+	jsonRsp := getSTHResponse{
```

Comment:
I have quite a strong preference for `req` / `rsp` as a pair (from long habit), and `json` alone would clash with the package name.